ffi/targets.cpp

```diff
@@ -194,18 +194,24 @@ LLVMPY_CreateTargetMachine(LLVMTargetRef T,
 
     CodeModel::Model cm;
     std::string cms(CodeModel);
-    if (cms == "jitdefault")
-        cm = CodeModel::JITDefault;
-    else if (cms == "small")
+    if (cms == "small")
         cm = CodeModel::Small;
     else if (cms == "kernel")
         cm = CodeModel::Kernel;
     else if (cms == "medium")
         cm = CodeModel::Medium;
     else if (cms == "large")
         cm = CodeModel::Large;
-    else
-        cm = CodeModel::Default;
+    else if (cms == "default") // As per LLVM 5, needed for AOT
```

Comment:
`As per LLVM 5` ?  Should it be LLVM 6?

Reply:
I think it should be `5` as `default` and `jitdefault` don't exist in 6, I added them back in as per the behaviour in 5.